Using MAPPING_STR=12121212, VERSION_DIR=v4_engman
Running /platform/source/s01/v4_engman/m12121212/m12121212_q10_t0.sh
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q11_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]            Get:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease [114 kB]
0% [2 InRelease 2687 B/114 kB 2%]0% [2 InRelease 14.3 kB/114 kB 13%]                                   0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
0% [Waiting for headers]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]100% [Working]              Fetched 114 kB in 1s (122 kB/s)
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-dev is already the newest version (3.8.2-0ubuntu2).
python3-dev set to manually installed.
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Collecting setuptools
  Downloading setuptools-69.0.2-py3-none-any.whl (819 kB)
[?25l[K     |‚ñç                               | 10 kB 49.8 MB/s eta 0:00:01[K     |‚ñâ                               | 20 kB 6.7 MB/s eta 0:00:01[K     |‚ñà‚ñè                              | 30 kB 9.8 MB/s eta 0:00:01[K     |‚ñà‚ñã                              | 40 kB 4.3 MB/s eta 0:00:01[K     |‚ñà‚ñà                              | 51 kB 4.6 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñç                             | 61 kB 5.4 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñâ                             | 71 kB 5.4 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñè                            | 81 kB 6.1 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñã                            | 92 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà                            | 102 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñç                           | 112 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñâ                           | 122 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                          | 133 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                          | 143 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                          | 153 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                         | 163 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                         | 174 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                        | 184 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                        | 194 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                        | 204 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                       | 215 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                       | 225 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                      | 235 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                      | 245 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                      | 256 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                     | 266 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                     | 276 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                    | 286 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                    | 296 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                    | 307 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                   | 317 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                   | 327 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                  | 337 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                  | 348 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                  | 358 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç                 | 368 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ                 | 378 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè                | 389 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã                | 399 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà                | 409 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç               | 419 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ               | 430 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè              | 440 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã              | 450 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà              | 460 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç             | 471 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ             | 481 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè            | 491 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã            | 501 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà            | 512 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç           | 522 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ           | 532 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè          | 542 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã          | 552 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà          | 563 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç         | 573 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ         | 583 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè        | 593 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã        | 604 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà        | 614 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç       | 624 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ       | 634 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè      | 645 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã      | 655 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà      | 665 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç     | 675 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ     | 686 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè    | 696 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã    | 706 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà    | 716 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç   | 727 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ   | 737 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè  | 747 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã  | 757 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà  | 768 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñç | 778 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñâ | 788 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñè| 798 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñã| 808 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 819 kB 6.8 MB/s eta 0:00:01[K     |‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 819 kB 6.8 MB/s 
[?25hInstalling collected packages: setuptools
  Attempting uninstall: setuptools
    Found existing installation: setuptools 45.2.0
    Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
    Can't uninstall 'setuptools'. No files were found to uninstall.
Successfully installed setuptools-69.0.2
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q12_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3 set to manually installed.
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q13_t0.sh
/platform/source/s01/v4_engman/m12121212/m12121212_q13_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engman/m12121212/m12121212_q13_t0.sh: line 7: sudo: command not found
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q14_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q15_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q16_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Collecting git+https://github.com/RedisLabsModules/direct_redis.git
  Cloning https://github.com/RedisLabsModules/direct_redis.git to /tmp/pip-req-build-4iifo2lo
  Running command git clone -q https://github.com/RedisLabsModules/direct_redis.git /tmp/pip-req-build-4iifo2lo
[31m  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git clone -q https://github.com/RedisLabsModules/direct_redis.git /tmp/pip-req-build-4iifo2lo[0m
[31mERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?[0m
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q17_t0.sh
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Collecting git+https://github.com/panosc-eu/direct_redis.git
  Cloning https://github.com/panosc-eu/direct_redis.git to /tmp/pip-req-build-8_xylcue
  Running command git clone -q https://github.com/panosc-eu/direct_redis.git /tmp/pip-req-build-8_xylcue
[31m  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git clone -q https://github.com/panosc-eu/direct_redis.git /tmp/pip-req-build-8_xylcue[0m
[31mERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?[0m
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q18_t0.sh
/platform/source/s01/v4_engman/m12121212/m12121212_q18_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engman/m12121212/m12121212_q18_t0.sh: line 7: sudo: command not found
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q19_t0.sh
/platform/source/s01/v4_engman/m12121212/m12121212_q19_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4_engman/m12121212/m12121212_q19_t0.sh: line 8: sudo: command not found
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q1_t0.sh
/platform/source/s01/v4_engman/m12121212/m12121212_q1_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engman/m12121212/m12121212_q1_t0.sh: line 7: sudo: command not found
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Collecting git+https://github.com/amyangfei/direct-redis.git#subdirectory=python
  Cloning https://github.com/amyangfei/direct-redis.git to /tmp/pip-req-build-gsbcms0s
  Running command git clone -q https://github.com/amyangfei/direct-redis.git /tmp/pip-req-build-gsbcms0s
[31m  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git clone -q https://github.com/amyangfei/direct-redis.git /tmp/pip-req-build-gsbcms0s[0m
[31mERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?[0m
Running /platform/source/s01/v4_engman/m12121212/m12121212_q20_t0.sh
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q21_t0.sh
Running /platform/source/s01/v4_engman/m12121212/m12121212_q22_t0.sh
/platform/source/s01/v4_engman/m12121212/m12121212_q22_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engman/m12121212/m12121212_q22_t0.sh: line 5: sudo: command not found
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: numpy in /usr/local/lib/python3.8/dist-packages (1.24.4)
chmod: cannot access 'install_dependencies.sh': No such file or directory
Running /platform/source/s01/v4_engman/m12121212/m12121212_q2_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q3_t0.sh
/platform/source/s01/v4_engman/m12121212/m12121212_q3_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engman/m12121212/m12121212_q3_t0.sh: line 7: sudo: command not found
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q4_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
0% [Waiting for headers]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q5_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q6_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
Package python-pip is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python3-pip

E: Package 'python-pip' has no installation candidate
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q7_t0.sh
/platform/source/s01/v4_engman/m12121212/m12121212_q7_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engman/m12121212/m12121212_q7_t0.sh: line 7: sudo: command not found
/usr/bin/pip3:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q8_t0.sh
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engman/m12121212/m12121212_q9_t0.sh
/usr/bin/pip
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Collecting git+https://github.com/pandas-dev/pandas.git
  Cloning https://github.com/pandas-dev/pandas.git to /tmp/pip-req-build-hq4p2tan
  Running command git clone -q https://github.com/pandas-dev/pandas.git /tmp/pip-req-build-hq4p2tan
[31m  ERROR: Error [Errno 2] No such file or directory: 'git' while executing command git clone -q https://github.com/pandas-dev/pandas.git /tmp/pip-req-build-hq4p2tan[0m
[31mERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?[0m
/usr/bin/pip:6: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  from pkg_resources import load_entry_point
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q10_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q10_t0.csv /platform/expected/s01/q10.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q10_t0.py", line 25, in <module>
    customer_df = pd.read_json(redis_client.get('customer'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q11_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q11_t0.csv /platform/expected/s01/q11.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q11_t0.py", line 18, in <module>
    supplier_data = eval(redis_client.get('supplier'))
TypeError: eval() arg 1 must be a string, bytes or code object
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q12_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q12_t0.csv /platform/expected/s01/q12.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q12_t0.py", line 22, in <module>
    lineitem_df = pd.read_json(redis_client.get('lineitem'), orient='records')
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q13_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q13_t0.csv /platform/expected/s01/q13.csv
>>> Expected ((37, 2))
    CUSTDIST  C_COUNT
0       5000        0
1        676       10
2        651        9
3        618       11
4        554       12
5        548        8
6        514       13
7        487        7
8        485       19
9        461       18
10       454       14
11       444       20
12       442       16
13       438       17
14       430       15
15       396       21
16       378       22
17       355        6
18       322       23
19       262       24
20       188       25
21       184        5
22       162       26
23       138       27
24       103       28
25        92        4
26        59       29
27        49        3
28        29       30
29        26       31
30        19       32
31        12        2
32         8       33
33         7       34
34         5       35
35         3        1
36         1       36

>>> Actual ((33, 2))
    number_of_customers  number_of_orders
0                    18                 1
1                    51                 2
2                   164                 3
3                   361                 4
4                   525                 5
5                   684                 6
6                   757                 7
7                   764                 8
8                   732                 9
9                   664                10
10                  607                11
11                  594                12
12                  544                13
13                  578                14
14                  506                15
15                  513                16
16                  450                17
17                  383                18
18                  324                19
19                  244                20
20                  195                21
21                  127                22
22                   77                23
23                   65                24
24                   35                25
25                   15                26
26                    8                27
27                    7                28
28                    2                29
29                    1                30
30                    1                31
31                    1                32
32                    1                34

Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q14_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q14_t0.csv /platform/expected/s01/q14.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q14_t0.py", line 26, in <module>
    lineitem_df = pd.read_json(lineitem_str)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q15_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q15_t0.csv /platform/expected/s01/q15.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q15_t0.py", line 13, in <module>
    df_supplier = pd.read_json(redis_client.get('supplier'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q16_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q16_t0.csv /platform/expected/s01/q16.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q16_t0.py", line 30, in <module>
    supplier_df = pd.read_msgpack(r.get('supplier'))
AttributeError: module 'pandas' has no attribute 'read_msgpack'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q17_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q17_t0.csv /platform/expected/s01/q17.csv
>>> Expected ((1, 1))
     AVG_YEARLY
0  31865.742857

>>> Actual ((1, 1))
   average_yearly_loss
0                  0.0

Head row similarity: 0.21052631578947367
Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q18_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q18_t0.csv /platform/expected/s01/q18.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q18_t0.py", line 25, in <module>
    customer_df = pd.read_json(redis_client.get('customer'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q19_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q19_t0.csv /platform/expected/s01/q19.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q19_t0.py", line 27, in <module>
    lineitem_df = pd.read_json(redis_client.get("lineitem"))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q1_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q1_t0.csv /platform/expected/s01/q1.csv
An error occurred: argument of type 'method' is not iterable
Score.fail_missing_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q20_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q20_t0.csv /platform/expected/s01/q20.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q20_t0.py", line 21, in <module>
    supplier_df = pd.read_json(redis_client.get('supplier').decode('utf-8'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'decode'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q21_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q21_t0.csv /platform/expected/s01/q21.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q21_t0.py", line 23, in <module>
    supplier_df = pd.DataFrame(eval(r.get('supplier')))
TypeError: eval() arg 1 must be a string, bytes or code object
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q22_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q22_t0.csv /platform/expected/s01/q22.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q22_t0.py", line 28, in <module>
    excluded_customers = set(orders_df['O_CUSTKEY'])
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 3761, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/indexes/range.py", line 349, in get_loc
    raise KeyError(key)
KeyError: 'O_CUSTKEY'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q2_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q2_t0.csv /platform/expected/s01/q2.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q2_t0.py", line 25, in <module>
    df_region = pd.read_json(region)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q3_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q3_t0.csv /platform/expected/s01/q3.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q3_t0.py", line 45, in <module>
    lineitem_df = pd.read_json(lineitem_df_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q4_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q4_t0.csv /platform/expected/s01/q4.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q4_t0.py", line 33, in <module>
    joined_df = orders_df.merge(lineitems_df, left_on='O_ORDERKEY', right_on='L_ORDERKEY')
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 9843, in merge
    return merge(
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 148, in merge
    op = _MergeOperation(
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 737, in __init__
    ) = self._get_merge_keys()
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 1221, in _get_merge_keys
    left_keys.append(left._get_label_or_level_values(lk))
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 1778, in _get_label_or_level_values
    raise KeyError(key)
KeyError: 'O_ORDERKEY'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q5_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q5_t0.csv /platform/expected/s01/q5.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q5_t0.py", line 47, in <module>
    orders = mongo_query()
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q5_t0.py", line 19, in mongo_query
    orders = orders.merge(asia_nations, how='inner', left_on='O_CUSTKEY', right_on='N_NATIONKEY')
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/frame.py", line 9843, in merge
    return merge(
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 148, in merge
    op = _MergeOperation(
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 737, in __init__
    ) = self._get_merge_keys()
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 1221, in _get_merge_keys
    left_keys.append(left._get_label_or_level_values(lk))
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 1778, in _get_label_or_level_values
    raise KeyError(key)
KeyError: 'O_CUSTKEY'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q6_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q6_t0.csv /platform/expected/s01/q6.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q6_t0.py", line 12, in <module>
    lineitem_df = pd.read_json(lineitem_data_raw)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q7_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q7_t0.csv /platform/expected/s01/q7.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q7_t0.py", line 22, in <module>
    supplier_df = pd.read_json(redis_client.get('supplier'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q8_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q8_t0.csv /platform/expected/s01/q8.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q8_t0.py", line 28, in <module>
    lineitem_df = get_redis_data(dr, 'lineitem')
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q8_t0.py", line 13, in get_redis_data
    data = r.get(key)
  File "/usr/local/lib/python3.8/dist-packages/direct_redis/direct_redis.py", line 22, in get
    encoded = super().get(key)
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 1579, in get
    return self.execute_command('GET', name)
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 875, in execute_command
    conn = self.connection or pool.get_connection(command_name, **options)
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 1185, in get_connection
    connection.connect()
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 552, in connect
    sock = self._connect()
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 578, in _connect
    for res in socket.getaddrinfo(self.host, self.port, self.socket_type,
  File "/usr/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
TypeError: getaddrinfo() argument 1 must be string or None
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engman/m12121212/m12121212_q9_t0.py /platform/output/s01/v4_engman/m12121212/m12121212_q9_t0.csv /platform/expected/s01/q9.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engman/m12121212/m12121212_q9_t0.py", line 19, in <module>
    supplier = pd.read_json(redis.get('supplier'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
